website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,57 @@
+---
+author: Vadim Chelyshov
+title: Towards better releases
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+As many of you might have noticed, the previous `0.11.0` release didn't go smoothly due to a number of issues that came to light only after the release was published.
+Some of them were quite critical and the only option to continue to work was to downgrade Metals until `0.11.1` arrived.
+
+We apologize for that!
+
+However, this post is not only about saying sorry.
+The reasons why it's happened weren't a complete surprise for us.
+In order to avoid such situations in the future we have taken upon us to take steps that will allow us to detect issues early.
+Some of these issues were quite obvious and we were aware of them for a while.
+
+There were two main improvements that were implemented since the last release:
+- Backpublish Scala versions support
+- Pre-release version for VSCode extension
+
+## Backpublish Scala versions support
+
+We haven't mentioned this in the release notes for `0.11.0`, but from that version we have changed the way Metals detect whether a specific version is supported and thanks to that we are now able to backpublish new Scala versions support.
```

Comment:
```suggestion
We haven't mentioned this in the release notes for `0.11.0`, but since that version we have changed the way Metals detects whether a specific version is supported and thanks to that we are now able to backpublish new Scala versions support.
```